src/parser/jobs/pld/modules/Gauge.tsx

```diff
@@ -0,0 +1,68 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import JOBS from 'data/JOBS'
+import {dependency} from 'parser/core/Module'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, Suggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+// Gauge numbers
+const GAUGE_GAIN_AMOUNT = 5
+const GAUGE_SPEND_AMOUNT = 50
+
+export default class Gauge extends CoreGauge {
+	static title = t('pld.gauge.title')`Oath Gauge Usage`
+
+	@dependency private suggestions!: Suggestions
+
+	private oathGauge = this.add(new CounterGauge({
+		chart: {label: 'Oath Gauge', color: JOBS.PALADIN.colour},
+	}))
+
+	private oathConsumers = [
+		ACTIONS.SHELTRON.id,
+		ACTIONS.INTERVENTION.id,
+		ACTIONS.COVER.id,
+	]
```

Comment:
More of a personal thing but I wonder if it wouldn't be simpler to just make this a map with the earn/spend costs built in so you can have a single hook and ref it. The WAR gauge is a good example of this.

Reply:
Not sure if simpler, but cleaner (?)... either way, a good idea. I'm changing that.